.github/workflows/custom_spell.json

```diff
@@ -3,7 +3,8 @@
     "*.drawio",
     "*/**/CHANGELOG.rst",
     "external/lanelet2_matching/**",
-    "openscenario/openscenario_utility/openscenario_utility/resources/*.xsd"
+    "openscenario/openscenario_utility/openscenario_utility/resources/*.xsd",
+    "test_runner/scenario_test_runner/scenario/spawn_relative_object_position.yaml"
```

Comment:
@yamacir-kit 
I wrote in the json file that "euclidian" is misspelled, but it seems that comments on json are not allowed. The comment itself is also treated as a misspelling. Is it OK if I don't comment?

![image](https://github.com/tier4/scenario_simulator_v2/assets/39142679/c9cb67db-a132-42b5-a701-1d911ea3971a)

Reply:
@HansRobo 
I have received a forward to you could you please your advice?

Reply:
close with https://github.com/tier4/scenario_simulator_v2/pull/1030/commits/8e6bca7532b8798a3e4fc8f2f0e34abbb3ee3f91

Reply:
thanks and add reason for disabling
https://github.com/tier4/scenario_simulator_v2/pull/1030/commits/9578caacd756c45529099cea657e8c1a02765861